src/provider/application.js

```diff
@@ -229,8 +238,8 @@ class Application extends EventEmitter {
   * @param {object} message - The message to send.
   */
   send(message) {
-    // Dont' send messages if not embedded
-    if (window.self === window.top) {
+    // Don't send messages if not embedded, unless a custom dispatch function was provided
+    if (!this.dispatchFunction && window.self === window.top) {
```

Comment:
skipping message sending should be an implementation detail of the dispatcher as I indicate below